### ISSUE TITLE
Sell and buy Shoop in one Shop

### ISSUE THREAD

**Issue by KlenerTeufel96** [snowgears/shopbugs#28]
First: I love your Plugin it is the best! We use this for our Server :)

My question is, could you program that player can make sell and buy shop in one shop like in Chestshop?

**Comment by Mustill0** [comment]
I agree dude! I was going to ask the same thing that would be very useful
